frappe/public/less/website.less

```diff
@@ -515,9 +515,6 @@ a.active {
 }
 
 @media (max-width: 767px) {
-	.visible-xs {
```

Comment:
Removing utility classes is not the correct way of fixing this. You should remove the usage of this class on the element rather.